packages/fxa-auth-server/lib/senders/emails/templates/passwordForgotOtp/includes.json

```diff
@@ -0,0 +1,6 @@
+{
+  "subject": {
+    "id": "password-forgot-otp-subject",
+    "message": "Your confirmation code"
```

Comment:
I think this is fine to land as is, but should also confirm with product that they don't want the code in subject line since we have others like that.

Reply:
Sounds good.

@vpomerleau or @vesta0 ?  Not sure if that was discussed originally.  What do you think?

Reply:
The subject line wasn't discussed, but I'd opt to keep the code out of the subject line so it's not visible from a lock screen, for example.

The subject line from the `recovery` template is "Reset your password", maybe we'd like to re-use that here?